_pages/docs.html

```diff
@@ -22,11 +22,12 @@ <h5 class="mt-0">PyHC Standards</h5>
 
   <div class="container mb-2">
     <div class="media">
-      <a href="https://zenodo.org/record/2537188#.XOXWVC2ZN24"><i class="fas fa-book fa-2x mr-3"></i></a>
+      <a href="{{ site.baseurl }}/docs/meeting_reports/"><i class="fas fa-book fa-2x mr-3"></i></a>
       <div class="media-body">
-        <h5 class="mt-0">2018 PyHC Meeting Report</h5>
-        The official meeting report from the 2018 community meeting.<br>
-        <a href="https://doi.org/10.5281/zenodo.2537188"><img src="https://zenodo.org/badge/DOI/10.5281/zenodo.2537188.svg" alt="DOI"></a>
+        <h5 class="mt-0">
+          <a href="{{ site.baseurl }}/docs/meeting_reports/">PyHC Biannual Meeting Reports</a>
+        </h5>
+        PyHC biannual meeting reports summarizing meeting details (i.e. participant list, meeting summary and final agenda, conclusions, and future considerations).<br>
```

Comment:
Wanna make sure you did mean "i.e." (in other words) rather than "e.g." (for example) here. If that's an exhaustive list of what's in the reports, then "i.e.", otherwise "e.g." 🤓

Reply:
Hmmmm. It is exhaustive, but who knows, we could add new things in the future! I'll make it e.g. 😅